### PR TITLE
Release v0.10.0

### LINKED ISSUE #127: Add PyPI classifiers
We should add [PyPI classifiers](https://pypi.org/classifiers/). At least the ones for Python version support.

We can check how uvicorn, gunicorn, fastapi, flask, etc. add to their classifiers.

### PR DESCRIPTION
**Pull request by sansyrox**
**Description**

This is a PR for the v0.10.0 release

Also, Fixes #127

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -24,5 +24,19 @@ dependencies = [
   'uvloop == 0.16.0; platform_machine == "x86_64"',
   'uvloop == 0.16.0; platform_machine == "i686"'
 ]
+classifiers = [
```
Comment (sansyrox) [inline pyproject.toml:27]: Hey @Kludex ,

I hope everything is well. 

I did add the Python Classifiers(taking inspiration from uvicorn and the pypi examples).

Can you have a look if you have anything to recommend/change in these?

File: pyproject.toml [hunk 2]
```diff
@@ -24,5 +24,19 @@ dependencies = [
...
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: BSD License",
+        "Operating System :: OS Independent",
+        "Topic :: Internet :: WWW/HTTP",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: Implementation :: CPython",
+        "Programming Language :: Python :: Implementation :: PyPy",
```
Comment (Kludex) [inline pyproject.toml:39]: Does it really supports PyPy? :thinking:

**Review by Kludex — COMMENTED** [review]
Just this one.

File: pyproject.toml (hunk 2 above)
Comment (sansyrox) [reply pyproject.toml:39]: Oh, right. It doesn't. I misread it as `PyPi`.

Thank you! :D

Comment (sansyrox) [reply pyproject.toml]: Updated.

**Pull request merged**